### PR TITLE
Bitcoin auto-mining when running locally, bump contracts to bring in changes required to allow testin with Xverse wallet locally

### PR DESCRIPTION
**Pull request by antonyshchenko**

### CODE REVIEW

File: terraform/testnet/main.tf [hunk 1]
```diff
@@ -107,14 +107,14 @@ module "waf" {
   aws_region = var.aws_region
   aws_lb_arn = module.alb.lb_arn
   #aws_lb_arn               = module.alb.https_listener_arn
-  short_env_name           = local.name_prefix
-  ipv4_blacklist_addresses = local.ipv4blacklist
-  ipv4_whitelist_addresses = local.ipv4whitelist
-  ipv6_blacklist_addresses = local.ipv6blacklist
-  ipv6_whitelist_addresses = local.ipv6whitelist
-  blocked_country_codes    = var.blocked_country_codes
+  short_env_name                = local.name_prefix
```
Comment (antonyshchenko) [inline terraform/testnet/main.tf:110]: Result of `make format`

File: src/main/kotlin/xyz/funkybit/LocalMain.kt [hunk 2]
```diff
@@ -30,6 +31,7 @@ fun main(args: Array<String>) {
                 RingApp().start()
                 ApiApp().start()
                 BotApp().start()
+                BitcoinMiner.start()
```
Comment (antonyshchenko) [inline src/main/kotlin/xyz/funkybit/LocalMain.kt:34]: Without this Bitcoin deposits and settlements get stuck locally

File: backend/src/test/kotlin/xyz/funkybit/core/bitcoin/UtxoManagerTest.kt [hunk 3]
```diff
@@ -311,7 +311,7 @@ class UtxoManagerTest : TestWithDb() {
             ),
         )
 
-        every { MempoolSpaceClient.getTransactions(feePayerAddress, txId4) } returns listOf()
+        every { MempoolSpaceClient.getTransactions(feePayerAddress, txId1) } returns listOf()
```
[the remark this answers is not in the document]
Comment (bflood366) [reply backend/src/test/kotlin/xyz/funkybit/core/bitcoin/UtxoManagerTest.kt:314]: ah good catch

File: terraform/testnet/main.tf (hunk 1 above)
Comment (bflood366) [reply terraform/testnet/main.tf:110]: I think we had some terraform for saving fulcrum image in ECR - can be removed.

**Review by bflood366 — APPROVED** [review]
looks great

File: terraform/testnet/main.tf (hunk 1 above)
Comment (antonyshchenko) [reply terraform/testnet/main.tf:110]: yup, done

**Pull request merged**